### DISCUSSION TITLE
gridpoints forecast no longer returning 'Last-Modified' header

### DISCUSSION THREAD

**Discussion by will-camus** [weather-gov/api#529] [General]
The [FAQ](https://weather-gov.github.io/api/gridpoints) state that the APIs will return a `Last-Modified` HTTP header in a response, but this header seems to have disappeared as of 2022-02-07T11:00Z [ this is the last time we received data with that header ].

Will this be a permanent change in the API?

Edit: here's an example response for today:
```console
$ curl --silent 'https://api.weather.gov/gridpoints/STO/34,31' -I
HTTP/2 200
server: nginx/1.20.1
content-type: application/geo+json
access-control-allow-origin: *
access-control-expose-headers: X-Correlation-Id, X-Request-Id, X-Server-Id
x-request-id: f1e5e188-d020-4d92-bbcb-c94d66497a19
x-correlation-id: 27c68fe4
x-server-id: vm-lnx-nids-apiapp3.ncep.noaa.gov
cache-control: public, max-age=3492, s-maxage=3600
expires: Sat, 12 Feb 2022 00:49:46 GMT
date: Fri, 11 Feb 2022 23:51:34 GMT
x-edge-request-id: 6cbf8a7c
vary: Accept,Feature-Flags,Accept-Language
strict-transport-security: max-age=31536000 ; includeSubDomains ; preload
```

**Comment by chaunceygardiner** [comment]
It’s been temporarily removed.

See:
https://www.weather.gov/media/notification/pdf2/scn22-23_api_response_headers.pdf

**Reply by will-camus** [reply]
thanks so much for the quick response! I'll check outage postings before posting here next time. Do you know if there is there a directory/feed of postings/changes that I could subscribe to?

edit: other than all of the [top-level postings](https://www.weather.gov/notification/)

**Reply by chaunceygardiner** [reply]
No problem.  I posted the same question.  I’m just passing along the quick response that I received.

The notifications are here:
https://www.weather.gov/notification/
